x-pack/plugins/alerting/server/application/rule/methods/bulk_delete/bulk_delete.ts

```diff
@@ -8,27 +8,35 @@ import pMap from 'p-map';
 import { KueryNode, nodeBuilder } from '@kbn/es-query';
 import { SavedObjectsBulkUpdateObject } from '@kbn/core/server';
 import { withSpan } from '@kbn/apm-utils';
-import { RawRule } from '../../types';
-import { convertRuleIdsToKueryNode } from '../../lib';
-import { bulkMarkApiKeysForInvalidation } from '../../invalidate_pending_api_keys/bulk_mark_api_keys_for_invalidation';
-import { ruleAuditEvent, RuleAuditAction } from '../common/audit_events';
-import { tryToRemoveTasks } from '../common';
-import { API_KEY_GENERATE_CONCURRENCY } from '../common/constants';
+import { RawRule } from '../../../../types';
+import { convertRuleIdsToKueryNode } from '../../../../lib';
+import { bulkMarkApiKeysForInvalidation } from '../../../../invalidate_pending_api_keys/bulk_mark_api_keys_for_invalidation';
+import { ruleAuditEvent, RuleAuditAction } from '../../../../rules_client/common/audit_events';
+import { tryToRemoveTasks } from '../../../../rules_client/common';
+import { API_KEY_GENERATE_CONCURRENCY } from '../../../../rules_client/common/constants';
 import {
   getAuthorizationFilter,
   checkAuthorizationAndGetTotal,
   getAlertFromRaw,
   migrateLegacyActions,
-} from '../lib';
+} from '../../../../rules_client/lib';
 import {
   retryIfBulkOperationConflicts,
   buildKueryNodeFilter,
-  getAndValidateCommonBulkOptions,
-} from '../common';
-import { BulkOptions, BulkOperationError, RulesClientContext } from '../types';
-
-export const bulkDeleteRules = async (context: RulesClientContext, options: BulkOptions) => {
-  const { ids, filter } = getAndValidateCommonBulkOptions(options);
+} from '../../../../rules_client/common';
+import type { RulesClientContext } from '../../../../rules_client/types';
+import type {
+  BulkOperationErrorV1,
+  BulkDeleteRulesResponseV1,
+  BulkDeleteRulesRequestParamsV1,
```

Comment:
Since this is the application layer, we don't need to version anything here. So we can just import the un-versioned types and schemas.

Reply:
2410993ae1d6047cbb272d1af947aa0e8b1acb85